implementations/elixir/ockam/ockam/test/ockam/secure_channel/key_establishment_protocol/xx/protocol_test.exs

```diff
@@ -1,103 +1,4 @@
 defmodule Ockam.SecureChannel.KeyEstablishmentProtocol.XX.Protocol.Tests do
   use ExUnit.Case, async: true
   doctest Ockam.SecureChannel.KeyEstablishmentProtocol.XX.Protocol
-
-  alias Ockam.SecureChannel.KeyEstablishmentProtocol.XX.Protocol
```

Comment:
We definitely can't remove them. We should either keep them, or bring them back soon after the merge

Reply:
These vectors where from public ones that don't work anymore due to the protocol name change.   You mean to setup vectors for whatever we produce now,  or make an option to use the previous  name and use that option for this test?